.goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+# This is an goreleaser.yaml file which builds mongodb_exporter releases.
+# Make sure to check the documentation at http://goreleaser.com
+# You can run it locally or with your CI, but you should use `make community-release` or `make snapshot`
+# instead of using `goreleaser` directly as some init steps should be done before it runs.
+dist: bin
+builds:
+  - binary: mongodb_exporter
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - netbsd
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: netbsd
+        goarch: arm
+      - goos: darwin
+        goarch: arm64
+      - goos: netbsd
+        goarch: arm64
+    flags:
+      - -v
+      - -a
+      - -tags=netgo
+    # We don't use internal `GoReleaser` variables because of consistency with `Makefile` and
+    # because we can't change it behaviour.
+    ldflags: |
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.ProjectName={{.Env.BIN_NAME}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.Version={{.Env.APP_VERSION}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.PMMVersion={{.Env.PMM_RELEASE_VERSION}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.Timestamp={{.Env.PMM_RELEASE_TIMESTAMP}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.FullCommit={{.Env.PMM_RELEASE_FULLCOMMIT}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/percona/pmm/version.Branch={{.Env.PMM_RELEASE_BRANCH}}
+        -X {{.Env.GO_PACKAGE}}/vendor/github.com/prometheus/common/version.BuildUser={{.Env.USER}}@{{.Env.TRAVIS_APP_HOST}}
```

Comment:
Maybe we could remove "github.com/prometheus/common/version"? Is it used?

Reply:
It used here: 
https://github.com/percona/mongodb_exporter/blob/PMM-3513-goreleaser/mongodb_exporter.go#L102

It's all about PMM-3250. Maybe we should add this helpers to our shared code (in `github.com/percona/pmm/version` package)?

Reply:
Or to https://github.com/percona/exporter_shared … Ok, let's keep it for now.